inst/@symfun/isequal.m

```diff
@@ -0,0 +1,96 @@
+%% Copyright (C) 2017 NVS Abhilash
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @documentencoding UTF-8
+%% @defmethod  @@symfun isequal (@var{f}, @var{g})
+%% @defmethodx @@symfun isequal (@var{f}, @var{g}, @dots{})
+%% Test if contents of two or more arrays are equal.
+%%
+%% Example:
+%% @example
+%% @group
+%% syms x
+%% f(x) = x + 1;
+%% g(x) = x + 1;
+%% isequal(f, g)
+%%   @result{} 1
+%% @end group
+%% @end example
+%%
+%% Note: two symfuns are equal if they have the same formula @emph{and}
+%% same arguments:
+%% @example
+%% @group
+%% syms x y
+%% f(x) = x + 1;
+%% g(x, y) = x + 1;
+%% isequal(f, g)
+%%   @result{} 0
+%% @end group
+%% @end example
+%%
+%% @seealso{@@sym/isequal, @@symfun/formula, @@symfun/argnames}
+%% @end defmethod
+
+function t = isequal(x, y, varargin)
+
+  t = logical(1);
+  if isa (x, 'symfun') && isa (y, 'symfun')
+    t = isequal (argnames(x), argnames(y));
```

Comment:
Argnames should work for sym too, return empty, so do we need this isa symfun check?